Sources/Gui/SDLRunner.cpp

```diff
@@ -45,7 +45,11 @@
 SPADES_SETTING(r_videoWidth);
 SPADES_SETTING(r_videoHeight);
 DEFINE_SPADES_SETTING(r_fullscreen, "0");
+#ifdef __linux__
+DEFINE_SPADES_SETTING(r_vsync, "-1");
+#else
```

Comment:
Have you verified the impact of defaulting `r_vsync` to `-1`? The failure of `SDL_GL_SetSwapInterval` is fatal, so I would rather play safe with this.

Reply:
yes see below.

Reply:
I'm not sure if you understood me. What I meant was: Are you sure the new default value `-1` is safe to use in most Linux systems? I'd hate to see new users being puzzled by an error message saying "SDL_GL_SetSwapInterval failed".

Reply:
ah ok I only tried two distros, good point, I ll revert it then